<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationControlWidget</class>
 <widget class="QDockWidget" name="AnimationControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>306</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Animation Control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>400</number>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Bones</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="boneList">
          <property name="modelColumn">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="minSliderLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="AnimationControlSlider" name="horizontalSlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="maxSliderLabel">
            <property name="text">
             <string>--</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>430</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>430</width>
              <height>110</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>90</number>
            </attribute>
            <row>
             <property name="text">
              <string>Translation</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Scale</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Rotation</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>w</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>x</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>y</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>z</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>-</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>-</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>0</string>
             </property>
             <property name="flags">
              <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationControlSlider</class>
   <extends>QSlider</extends>
   <header>animationcontrolslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
